rocky/rocky/scheduler.py

```diff
@@ -159,16 +163,23 @@ def __getitem__(self, key) -> List[Task]:
         return res.results
 
 
-class TooManyRequestsError(Exception):
-    pass
+class SchedulerError(Exception):
+    message = _("Connectivity issues with Mula.")
```

Comment:
## Checklist for QA:
- [x] I have checked out this branch, and successfully ran a fresh `make reset`.
- [x] I confirmed that there are no unintended functional regressions in this branch:
    - [x] I have managed to pass the onboarding flow
    - [x] Objects and Findings are created properly
    - [x] Tasks are created and completed properly
- [x] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.

### What works:
- Proper "queue is full" messages for both boefjes and normalizers
- Proper duplicate task messages for both boefjes and normalizers
- No 500 errors no matter how much I try to overload Rocky and Mula
 - No apparent regressions

### Bug or feature?:
* There is still a 500 error when the scheduler is down. In this piece of code, there is a general exception handler. Maybe this could be extended to account for a `ConnectionError`?